Library/Homebrew/formula.rb

```diff
@@ -1288,9 +1290,18 @@ def system(cmd, *args)
           end
           wr.close
 
-          while buf = rd.gets
-            log.puts buf
-            puts buf
+          if verbose
+            while buf = rd.gets
+              log.puts buf
+              puts buf
+            end
+          else # travis
```

Comment:
I really don't like the idea of making `system` behave differently for just one CI system.